From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jianfeng Liu <Jianfeng.Liu@cixtech.com>
Date: Wed, 25 Mar 2026 11:36:26 +0800
Subject: Revert "use afbc as default format"

AFBC format is not supported by mainline kernel, so change the default
format back to NV12M.
---
 driver/if/v4l2/mvx_v4l2_fops.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/driver/if/v4l2/mvx_v4l2_fops.c b/driver/if/v4l2/mvx_v4l2_fops.c
index c881551b92c0..2ce78a562cab 100755
--- a/driver/if/v4l2/mvx_v4l2_fops.c
+++ b/driver/if/v4l2/mvx_v4l2_fops.c
@@ -75,7 +75,7 @@ int mvx_v4l2_open(struct file *file)
 
     /* Set default port formats. */
     fmt.type = V4L2_BUF_TYPE_VIDEO_OUTPUT_MPLANE;
-    fmt.fmt.pix.pixelformat = V4L2_PIX_FMT_YUV420_AFBC_8;
+    fmt.fmt.pix.pixelformat = V4L2_PIX_FMT_NV12M;
     fmt.fmt.pix.width = 64;
     fmt.fmt.pix.height = 64;
     (void)mvx_v4l2_vidioc_s_fmt_vid_out(file, NULL, &fmt);
-- 
2.53.0

